docs/docs/troubleshooting.md

```diff
@@ -1934,8 +2034,6 @@ Since Ducktape is inspired by Chimney, there is a huge overlap in functionality.
     * customizing the [field-](supported-transformations.md#customizing-field-name-matching) and
       [subtype-name](supported-transformations.md#customizing-subtype-name-matching) matching methods
     * [sharing flags overrides between all derivations in the same scope](cookbook.md#reusing-the-flags-for-several-transformationspatchings)
-    * [smart constructors](supported-transformations.md#types-with-manually-provided-constructors), not only custom
```

Comment:
Dropping this point since ducktape also supports smart cosntructors (https://github.com/arainko/ducktape/blob/456df5656e223ebab9bcc044f65230481591b6ec/ducktape/src/main/scala/io/github/arainko/ducktape/syntax.scala#L107) - I'll have a follow up PR that fills this in the docs as well :smile: